addons/website/tests/test_widgets.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import HttpCase, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestWidgets(HttpCase):
+
+    def test_widget_lifecycle(self):
+        self.env['ir.asset'].create({
+            'name': 'countdown_patch',
+            'bundle': 'web.assets_frontend',
+            'path': 'website/static/tests/tour_utils/widget_lifecycle_dep_widget.js',
+        })
```

Comment:
I think you can put that one in the test bundle instead no? It's nice as you did but it may slow down the runbot so best avoid it for this one.